go/README.md

```diff
@@ -4,27 +4,70 @@ This builder runs the `go` tool (`go build`, `go test`, etc.)
 after placing source in `/workspace` into the `GOPATH` before
 running the tool.
 
-This functionality is not necessary if you're building using
+# Using `Golang` and [Go Modules](https://github.com/golang/go/wiki/Modules)
```

Comment:
Docker image names must be lowercase, so I'd prefer to refer to it as `golang`

Reply:
Fixed: https://github.com/GoogleCloudPlatform/cloud-builders/pull/517/commits/f7fbd8a23bdb6a3be7e5911df90506958a525852